mail_environment/README.rst

```diff
@@ -0,0 +1,113 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+mail environment
+================
+
+This module allows to configure the incoming and outgoing mail servers
+using the server_environment mechanism.
+
+Installation
+============
+
+To install this module, you need to have the server_environment module
+installed and properly configured.
+
+Configuration
+=============
+
+With this module installed, the incoming and outgoing mail servers are
+in the server_environment_files module that you should provide.
+
+In the configuration file for each environment, you may use the
```

Comment:
In the configuration file of each environment, you may use the
sections 'outgoing_mail' and 'incoming_mail' to configure the default
values respectively for SMTP servers and the IMAP/POP servers.